fineract-progressive-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/calc/ProgressiveEMICalculator.java

```diff
@@ -193,6 +193,7 @@ void calculateEMIValueAndRateFactors(final LocalDate calculateFromRepaymentPerio
         calculateOutstandingBalance(scheduleModel);
         calculateEMIOnPeriods(relatedRepaymentPeriods, scheduleModel);
         calculateOutstandingBalance(scheduleModel);
+        adjustRateFactor(scheduleModel);
```

Comment:
I think we can move this inside `calculateRateFactorForPeriods`... What do you think?

Reply:
Ok, will check it